packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -737,6 +743,7 @@ describe('Launchpad: Setup Project', () => {
           cy.contains('cypress.config.ts')
           cy.containsPath('cypress/component/index.html')
           cy.containsPath(`cypress/support/component.ts`)
+          cy.containsPath(`cypress/support/commands.ts`)
```

Comment:
Can we take this chance to add some tests to actually verify the file is **created** on the filesystem? These tests just verify we show the file we will create in the UI, not that they are actually written.

We have a pattern for this, see this function: https://github.com/cypress-io/cypress/blob/10.0-release/packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts#L11-L19 . This one actually verifies the content, too (might be worth doing something similar here?)